LICENSE.md

```diff
@@ -22,39 +22,51 @@ with this program.  If not, see <https://gno.land/license>.
-user who uses or modifies this software, the owner and maintainers of primary
-portals, and the validators and full nodes that run the blockchain software may
-all be different entities, but the strong attribution clause shall persist
-across these entities whereever contractional relationships exist.
+
+# Additional Terms (Strong Attribution)
+
+In accordance with Section 7 of the License, any user interface of a Covered
+Work, or of any other work or service that does not include the Covered Work
+but which derives functionality from it or otherwise uses it (an “Applicable
+Work”), and any material that promotes the Covered Work, must prominently
+display a reasonable attribution notice for the Covered Work (the “Attribution
+Notice”). The Attribution Notice shall, at minimum, include an active hyperlink
+to a URL designated by the Licensor for attribution (the “Attribution URL”),
+and up to three words such as "built on" or "forked from" as designated by the
+Licensor according to the Licensor's Attribution Policy.
+
+The Attribution Notice must be in a manner readily visible to users and must be
+at least as conspicuous as other legal notices or credits in the same
+interface. If there are no other notices or credits it is sufficient for the
+Attribution Notice to appear prominently in the header or footer of the
+interface. The Attribution URL and related attribution terms described herein
+are determined exclusively by NewTendermint, LLC. In cases of ambiguity,
+forks, or disputes regarding attribution, the designation made by
+NewTendermint, LLC shall conclusively govern.
+
+This attribution requirement remains in force for seven (7) years from the date
+the Covered Work is first publicly released under this License, or seven (7)
+years from the date you first make an Applicable Work available to the public,
+whichever is later. In no event shall this requirement continue beyond twelve
+(12) years from the Covered Work’s initial public release. If a court of
+competent jurisdiction or the Licensor finds that the designated Attribution
+URL violates users’ reasonable expectations of privacy or security, then the
+requirement to include that URL may be omitted or modified to the minimum
+extent necessary to avoid such issue. In all other cases, the attribution must
+be preserved as stated.
+
+You must preserve and display the Attribution Notice as required above in every
+Covered Work and Applicable Work that you propagate, operate, or otherwise make
+available to users or third parties. If you convey, deploy or promote the
+Covered Work to any third party, or permit or enable a third party to deploy,
+operate or promote an Applicable Work, you must require that such third party,
+as a condition of use, also preserve and maintain the same Attribution Notice
+and comply with the above attribution requirement. The attribution requirement
+is a condition of the License that travels with the Covered Work forever and
```

Comment:
The Additional (Strong Attribution) Term lives forever, though its effect is limited in time.